prep-html-skeleton/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML-Skeleton-Exercise</title>
+      <body>
+        <h1>I built a proper HTML skeleton!</h1>
+      </body>
```

Comment:
Be sure to make your `<body>` a sibling of the `<head>` instead of a child of it

```html
  <head>
    <title>HTML-Skeleton-Exercise</title>
  </head>
  <body>
    <h1>I built a proper HTML skeleton!</h1>
  </body>
```